CPP_STYLE_GUIDE.md

```diff
@@ -1,9 +1,10 @@
 # C++ style guide
 
-Unfortunately, the C++ linter (which can be run explicitly
-via `make cpplint`) does not currently catch a lot of rules that are specific
-to the Node.js C++ code base. This document explains the most common of these
-rules:
+Unfortunately, the C++ linter (based on
+[Google’s `cpplint`](https://github.com/google/styleguide)), which can be run
+explicitly via `make cpplint`, does not currently catch a lot of rules that are
```

Comment:
Should this perhaps be `make lint-cpp` instead? Running `make cppilnt` target produces the following warning:
```console
$ make cpplint
Running C++ linter...
Total errors found: 0
Please use lint-cpp instead of cpplint
```

Reply:
@danbev yes, I guess so :)